scripts/features/rabbitmq.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+# Check If rabbitmq Has Been Installed
+
+if [ -f /home/vagrant/.homestead-features/rabbitmq ]
+then
+    echo "rabbitmq already installed."
+    exit 0
+fi
+
+touch /home/vagrant/.homestead-features/rabbitmq
+
+
+# Install Erlang
+
+wget -q -O- https://packages.erlang-solutions.com/ubuntu/erlang_solutions.asc | sudo apt-key add -
+echo "deb https://packages.erlang-solutions.com/ubuntu bionic contrib" | sudo tee /etc/apt/sources.list.d/rabbitmq.list
+sudo apt-get update
+sudo apt-get -y install erlang
+
+# Install RabbitMQ
+
+wget -q -O- https://dl.bintray.com/rabbitmq/Keys/rabbitmq-release-signing-key.asc | sudo apt-key add -
+wget -q -O- https://www.rabbitmq.com/rabbitmq-release-signing-key.asc | sudo apt-key add -
+echo "deb https://dl.bintray.com/rabbitmq/debian $(lsb_release -sc) main" | sudo tee /etc/apt/sources.list.d/rabbitmq.list
+sudo apt-get update
```

Comment:
We should group the statements we need to do *before* `apt-get update` so we only do the update once. This will save time during provisioning.

Reply:
You are right. That's now corrected.